requirements.txt

```
@@ -13,6 +13,7 @@ signac-dashboard==0.3.1
 # commonly used packages
 h5py==3.7.0
 jupyter==1.0.0; platform_machine != 'ppc64le'
+jupyterlab==1.0.10
```

```suggestion
jupyterlab==3.5.0; platform_machine != 'ppc64le'
```
Install the latest version and exclude it on ppc64le systems.